lib/adal.js

```diff
@@ -822,8 +893,8 @@ var AuthenticationContext = (function () {
 
     /**
      * Gets resource for given endpoint if mapping is provided with config.
-     * @param {string} endpoint  -  API endoibt
-     * @returns {string} resource for this API endpoint
+     * @param {string} endpoint  -  API endpoint
```

Comment:
Can we change the description to: The uri for which the resource Id is requested.